src/display/annotation_layer.js

```diff
@@ -290,12 +290,14 @@ class AnnotationElement {
 
 class LinkAnnotationElement extends AnnotationElement {
   constructor(parameters) {
-    const isRenderable = !!(
+    const _isRenderable = !!(
       parameters.data.url ||
       parameters.data.dest ||
-      parameters.data.action
+      parameters.data.action ||
+      parameters.data.isTooltipOnly
     );
-    super(parameters, isRenderable);
+
```

Comment:
Same here, we can revert the changes in this line and the next one to get a smaller diff.